modules/storages/spec/features/storages_module_disabled_spec.rb

```diff
@@ -59,13 +59,6 @@
     Capybara.raise_server_errors = old_value
   end
 
-  it 'removes storages from administration and project pages' do
```

Comment:
@apfohl Removing these lines makes the whole file not do anything anymore, I believe. So we either remove it completely or we find stuff in here, we want to have covered even without the feature flag present. For example convering the consequences of activating and disabling the File storages module per project.

Reply:
I left the part in there that test activated case.

Reply:
I refactored the spec and added a disabled module spec.